websites/D/Desu-Online/presence.ts

```diff
@@ -0,0 +1,159 @@
+const presence = new Presence({
+		clientId: "580032576434077707",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video = {
+	current: 0,
+	duration: 0,
+	paused: true,
+};
+
+enum image {
+	search = "https://i.imgur.com/1VwDv4D.png",
+	home = "https://i.imgur.com/xVhTZcB.png",
+	play = "https://i.imgur.com/Zd2WTR1.png",
+	pause = "https://i.imgur.com/LV9xIjD.png",
+	logo = "https://i.imgur.com/KEkxfKO.png",
```

Comment:
Capitals. Also change the mentions of this.
```suggestion
	Search = "https://i.imgur.com/1VwDv4D.png",
	Home = "https://i.imgur.com/xVhTZcB.png",
	Play = "https://i.imgur.com/Zd2WTR1.png",
	Pause = "https://i.imgur.com/LV9xIjD.png",
	Logo = "https://i.imgur.com/KEkxfKO.png",
```